tools/please_go/install/toolchain/toolchain_test.go

```diff
@@ -0,0 +1,16 @@
+package toolchain
+
+import (
+	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestGetVersion(t *testing.T) {
+	tc := &Toolchain{GoTool: filepath.Join(os.Getenv("DATA"), "bin/go")}
+
+	ver, err := tc.GoMinorVersion()
+	require.NoError(t, err)
+	require.Equal(t, 17, ver)
```

Comment:
This will fail with every toolchain upgrade.

Reply:
Yeah but that's fairly rare and easy to fix :)